packages/frontend/src/components/Layout/index.tsx

```diff
@@ -75,7 +75,7 @@ export default function Layout({
           onClose={closeDrawer}
         />
 
-        <Box sx={{ flex: 1 }}>{children}</Box>
+        <Box sx={{ flex: 1, overflowX: 'hidden' }}>{children}</Box>
```

Comment:
comment:

🤔 I'm surprised this didn't break anything, but looks like we're responsive enough to not require horizontal scroll in our non-tiles screens.